dbms/src/DataStreams/FilterBlockInputStream.cpp

```diff
@@ -67,12 +67,13 @@ Block FilterBlockInputStream::readImpl()
     /// Until non-empty block after filtering or end of stream.
     while (true)
     {
-        res = children.back()->read();
+        FilterPtr child_filter = nullptr;
+        res = children.back()->read(child_filter, true);
```

Comment:
Now `UnorderedInputStream` does not support `Block readImpl(FilterPtr & res_filter, bool return_filter)` and the default table scan stream is `UnorderedInputStream`.
Will `UnorderedInputStream` support `Block readImpl(FilterPtr & res_filter, bool return_filter)` in the future?

Reply:
yes, the child of `UnorderedInputStream` is `DMVersionFilterBlockInputStream` which already supports `Block readImpl(FilterPtr & res_filter, bool return_filter)`. So it is easy to implement, I will open another pr later.